public/javascripts/Gallery/src/modal/GalleryPanorama.js

```diff
@@ -50,15 +50,14 @@
             height: '60vh'
         })[0];
 
-        self.panoNotAvailable = $("<div id='pano-not-avail'>Oops, our fault but there is no longer imagery available " +
-            "for this label.</div>").css({
+        self.panoNotAvailable = $(`<div id='pano-not-avail'>${'common:errors.error'}</div>`).css({
```

Comment:
Make sure that this one uses i18next as well! We have lots of checks to make it so that this code should never be used, but if it does, we should have the translation set up right :ok_hand: